lib/server/parse-websocket.js

```diff
@@ -32,5 +33,15 @@ function parseWebSocketRequest (socket, params) {
   )
   params.compact = -1 // return full peer objects (used for websocket responses)
 
+  params.ip = opts.trustProxy
+      ? socket.upgradeReq.headers['x-forwarded-for'] || socket.upgradeReq.connection.remoteAddress
+      : (socket.upgradeReq.connection.remoteAddress && socket.upgradeReq.connection.remoteAddress.replace(common.REMOVE_IPV4_MAPPED_IPV6_RE, '')) // force ipv4
+  params.port = socket.upgradeReq.connection.remotePort
```

Comment:
I'd rather not name this `port` since `port` means something entirely different for the http server. There it means the client's reported torrent client port. There's no torrent client port for webrtc peers.